src/Paprika/Store/StorageFanOutPage.cs

```diff
@@ -102,6 +102,32 @@ public void Report(IReporter reporter, IPageResolver resolver, int level)
             }
         }
     }
+    public Page Destroy(IBatchContext batch, in NibblePath prefix)
+    {
+        if (Header.BatchId != batch.BatchId)
+        {
+            // the page is from another batch, meaning, it's readonly. Copy
+            var writable = batch.GetWritableCopy(page);
+            return new StorageFanOutPage<TNext>(writable).Set(prefix, ReadOnlySpan<byte>.Empty, batch);
+        }
+
+        // Destroy the Id entry about it
+        // Set(prefix, ReadOnlySpan<byte>.Empty, batch);
```

Comment:
@Scooletz  I have commented out the Set call here. Wanted to confirm if I need to add it to the DeleteByPrefix method?

Reply:
On each level, if a page has a map, the map should have all the prefixes removed.

Reply:
Hey, does this point still need to be addressed?